s3prl/downstream/runner.py

```diff
@@ -72,15 +72,27 @@ def _init_model(self, model, name, trainable, interfaces=None):
 
 
     def _get_upstream(self):
-        Upstream = getattr(hub, self.args.upstream)
+        if "from_hf_hub" in self.args and self.args.from_hf_hub == True:
+            from huggingface_hub import snapshot_download
+
+            print(f'Downloading upstream model from the Hugging Face Hub {self.args.upstream}')
+            filepath = snapshot_download(self.args.upstream)
+            sys.path.append(filepath)
+
+            from expert import UpstreamExpert
+            Upstream = UpstreamExpert
+            ckpt_path = os.path.join(filepath, "model.pt")
```

Comment:
do i understand correctly that `model.pt` are the weights for the upstream model on the hub? 

what happens if the user submits weights with a different filename (e.g. `hubert.pt`)?

Reply:
Maybe this should be specified in the args as well, WDYT?

Reply:
yeah, sounds good!

Reply:
Done!